assets/src/common/helpers/index.js

```diff
@@ -55,9 +54,25 @@ export const hasMinimumDimensions = ( media, dimensions ) => {
  * @return {Object} Minimum dimensions including width and height.
  */
 export const getMinimumFeaturedImageDimensions = () => {
-	const width = MINIMUM_FEATURED_IMAGE_WIDTH;
+	const defaultDimensions = {
+		width: 1200,
+		height: 675,
+	};
+	/**
+	 * If settings not loaded in windows object, use default
+	 * values as fallback.
+	 */
+	if ( window.ampBlockEditor === undefined ||
+		window.ampBlockEditor.featuredImageMinimumWidth === undefined ||
+		window.ampBlockEditor.featuredImageMinimumHeight === undefined ) {
+		return defaultDimensions;
+	}
+	const height = parseInt( window.ampBlockEditor.featuredImageMinimumHeight );
+	const width = parseInt( window.ampBlockEditor.featuredImageMinimumWidth );
```

Comment:
This function could also be done a little more concisely with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (note the `?`s):

```JS
return {
	width: window.ampBlockEditor?.featuredImageMinimumWidth || 1200,
	height: window.ampBlockEditor?.featuredImageMinimumHeight || 675,
}
```